File: Valet Safe Android/Valet Safe Android/ValetSafe/UI/RegisterActivity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace Valet_Safe_Android.ValetSafe.UI
{
    [Activity(Label = "RegisterActivity")]
    public class RegisterActivity : Activity
    {

        private int userD = Resource.Drawable.register_user;
        private int passwordD = Resource.Drawable.register_password;
        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.register);
            EditText userEdit = FindViewById<EditText>(Resource.Id.UsernameEdit);
            EditText passwordEdit = FindViewById<EditText>(Resource.Id.PasswordEdit);
        }

        private void initDrawable()
        {


        }
    }
}

File: ValetSafeDroid/ValetSafeDroidDriver/app/src/main/java/com/android/valetsafe/valetsafedroid/RegisterActivity.java
package com.android.valetsafe.valetsafedroid;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.EditText;

public class RegisterActivity extends AppCompatActivity {
    private EditText name;
    private EditText phone;
    private EditText mail;
    private EditText password;
    private EditText repassword;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.register_layout);
        name = (EditText)findViewById(R.id.register_edit_name);
        phone = (EditText)findViewById(R.id.register_edit_phone);
        mail = (EditText)findViewById(R.id.register_edit_email);
        password = (EditText)findViewById(R.id.register_edit_password);
        repassword = (EditText)findViewById(R.id.register_edit_repassword);
    }
}
